### PR TITLE
Tdr content expand

### PR DESCRIPTION
**Pull request by pabloopez**

### CODE REVIEW

File: content/2-vulnerability-management/1-ecr_scanning/311_install_sysdig_ecr_scanner.md [hunk 1]
```diff
@@ -47,11 +40,16 @@ Check that the cron-job was properly installed with:
...
 
 ```
 NAME               SCHEDULE    SUSPEND   ACTIVE   LAST SCHEDULE   AGE
 registry-scanner   0 6 * * 6   False     0        <none>          2d1h
 ```
 
-In the next section we will find out how to trigger a scan action manually.
+In the next section we will find out how to:
+- Trigger a scan action manually.
+- And review the scan results in the Sysdig Secure UI.
```
Comment (manuelbcd) [inline content/2-vulnerability-management/1-ecr_scanning/311_install_sysdig_ecr_scanner.md]: May we remove this 'And'?

File: content/3-posture/1-compliance/identify_and_remediate.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Sysdig provides a general overview of your security posture.
+
+Go to Home
+
+Go to Posture > Compliance
+
+## Remediation for CSPM
+
+foo
```
Comment (manuelbcd) [inline content/3-posture/1-compliance/identify_and_remediate.md:18]: foo?

Comment (pabloopez) [reply content/3-posture/1-compliance/identify_and_remediate.md:18]: Render the current version, the Posture section is not indexed.

File: content/2-vulnerability-management/1-ecr_scanning/311_install_sysdig_ecr_scanner.md (hunk 1 above)
Comment (pabloopez) [inline content/2-vulnerability-management/1-ecr_scanning/311_install_sysdig_ecr_scanner.md]:
```suggestion
- Review the scan results in the Sysdig Secure UI.
```

Comment (pabloopez) [reply content/2-vulnerability-management/1-ecr_scanning/311_install_sysdig_ecr_scanner.md]: Done in [2a1026b](https://github.com/sysdiglabs/aws-modernization-with-sysdig/pull/22/commits/2a1026b97fd6fadb1784f649648d49fa147c97c4)

**Comment by manuelbcd** [comment]
Merged to main and deployed for preview to http://awsworkshop-review.s3-website-sa-east-1.amazonaws.com/ 
FYI @pabloopez @jeremiahsysdig

**Pull request merged**